enforcer-parent/enforcer/src/main/java/org/wso2/choreo/connect/enforcer/grpc/ExtAuthService.java

```diff
@@ -174,6 +177,9 @@ private CheckResponse buildResponse(CheckRequest request, ResponseObject respons
                 responseObject.getMetaDataMap().forEach((key, value) ->
                         structBuilder.putFields(key, Value.newBuilder().setStringValue(value).build()));
             }
+            structBuilder.putFields(RouterAccessLogConstants.ORIGINAL_PATH_DATA_NAME,
```

Comment:
So if the request is failed at enforcer level, we ask users to depend on the :path variable. And if it passes then we gonna depend on the OriginalPath metadata?
Just want to verify the behavior. IMO it is kinda okay since if it is an enforcer failure or 404 case, it anyway not gonna reach the enforcer.